benches/curve_factory_benchmarks.rs

```diff
@@ -0,0 +1,244 @@
+use std::{collections::HashMap, env, time::Duration};
+
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use futures::StreamExt;
+use num_bigint::BigUint;
+use rand::Rng;
+use tokio::runtime::Runtime;
+use tracing::{debug, info, warn};
+use tracing_subscriber::EnvFilter;
+use tycho_common::{models::token::Token, simulation::protocol_sim::ProtocolSim};
+use tycho_simulation::{
+    evm::{
+        engine_db::tycho_db::PreCachedDB,
+        protocol::{
+            filters::curve_pool_filter,
+            vm::state::EVMPoolState,
+        },
+        stream::ProtocolStreamBuilder,
+    },
+    protocol::models::ProtocolComponent,
+    tycho_client::feed::component_tracker::ComponentFilter,
+    tycho_common::models::Chain,
+    utils::load_all_tokens,
+};
+
+// Configuration constants and environment variables
+const DEFAULT_N_SWAPS: usize = 100;
+const DEFAULT_TVL_THRESHOLD: f64 = 1000.0;
+
+fn get_config() -> (usize, f64) {
```

Comment:
I find this function a bit strange.. it returns two variables. but where this function is called we are only using one or the other, never both at the same time. And also there are other env vars (a part of config as well) that are not here, like the tycho url and etc

Reply:
also this is repeated in both files 😬

Reply:
why not?